ParisBNF/abb/BNFabb186.xml

```diff
@@ -0,0 +1,733 @@
+                  <title type="incomplete" ref="LIT2765RepCh49">Malkǝʾ-hymn to Phanuel</title>
+                  <textLang mainLang="gez" />            
+                  <incipit xml:lang="gez">
+                     ሰላም፡ ለከ፡ ሰዳዴ፡ ሰይጣናት፡ ፋኑኤል፡ ለእግዚአብሔር፡ እምጽርኁ፡ ከመ፡ ኢይሰክዩ፡ 
+                  </incipit>
+                  <explicit xml:lang="gez">
+                     ብእሲ፡ ድኩም። <gap reason="illegible" unit="chars" quantity="3"/> ይጢ፡ ሕሙም። 
+                  </explicit>                                   
+               </msItem>
+               
+            </msContents>
+            <physDesc>
+               <objectDesc form="Codex">
+                  <supportDesc>
+                     <support>
+                        <material key="parchment">
+                        </material>
+                     </support>
+                     <extent>                        
+                        <measure unit="leaf"  quantity="1">1</measure>
+                        <locus from="13r" to="13v"/>                              
+                        <dimensions type="leaf" unit="mm">                                                         
+                        </dimensions>
+                     </extent>                          
+                  </supportDesc>
+                  
+                  
+                  <layoutDesc>
+                     <layout columns="1" writtenLines="25" xml:id="textarea5">                           
+                        <locus from="13r" to="13v"/>                             
+                     </layout>                        
+                  </layoutDesc>
+               </objectDesc>
+               
+               <handDesc>
+                  <handNote script="Ethiopic" xml:id="h5">       
+                     <locus from="13r" to="13v"/>              
+                     <seg type="script">19th-century script</seg>                     
+                     <desc>Irregular, careless hand; upright to slightly left sloping letters; 
+                        predominantly square, broad forms; by a mediocre scribe.</desc>
+                  </handNote>
+               </handDesc>              
+            </physDesc>
+            
+            <history>
+               <origin>
+                  <origDate notBefore="1750" notAfter="1868"/>
+               </origin>
+            </history>
+            </msPart>
+
+
+               <msPart xml:id="p6">
+                  <msIdentifier>
+                     <idno/>
+                  </msIdentifier>
+                  
+                  <msContents>
+                     <summary/>
+                     
+                     <msItem xml:id="p6_i1">
+                        <locus from="14r" to="37v"/>
+                        <title type="complete">Armenian printed booklet concerning calendar and computus</title>
+                        <textLang mainLang="ar" >Armenian</textLang>           
```

Comment:
Remove space in `<textLang mainLang="ar" >`  

I think p6_i1 is better to be defined as accMat.

Reply:
Yes, I though of this option for a while. A certain problem is that through the thorough foliation the print was incorporated into the structure of the manuscript (maybe already by d'Abbadie himself). If it would not have been the case (and if this print would have been loosely placed at th end, for instance), accMat would be perfect. It is not the case. I would leave it as it is, for now; but we can ask Zhenia what she thinks.

Reply:
I would opt to define it as accompanying material nevertheless, but both solutions might be possible.